scripts/analyze_failed_jobs/playbook.yml

```diff
@@ -0,0 +1,49 @@
+- hosts: build.clusterhq.com
+  become: true
+  vars: 
+    buildbot_dir: /var/lib/docker/vfs/dir/91ca9fb14be23372a04b364447224e6d8058dbac4ea51905c7c6e42fbb06f1bd
```

Comment:
Is this going to need to be changed after we redeploy the buildbot? If so, please add a comment.